src/main/webapp/WEB-INF/view/StylizedTextParser.java

```diff
@@ -0,0 +1,80 @@
+// StylizedTextParser.java
+// Class with methods to parse a message with markdown elements and
+// convert to HTML stylized text 
+
+//package codeu.controller;
+
+import java.lang.StringBuffer; 
+import java.lang.String; 
+import java.util.Stack; 
+import java.util.Hashtable; 
+
+public class StylizedTextParser {
+
+    /* Maps markdown symbol to array of HTML opening and closing stylized syntax */
+    private Hashtable<Character, String> styleTable; 
+
+    /* constructor */
+    public StylizedTextParser() {
+        styleTable = new Hashtable<Character, String>(); 
+        fill(styleTable); 
+    }
+
+    /* Fills styleTable with markdown elements to be parsed */
+    private static void fill(Hashtable<Character, String> styleTable) {
```

Comment:
I think you can hardcode all of this, unless you see a specific reason to programatically add these elements.